### ISSUE TITLE
Quill v2 Support

### ISSUE THREAD

**Issue by MentalGear** [reedsy/quill-cursors#91]
Thank you for this this! Does it also support quill version 2?

**Comment by alecgibson** [comment]
Yes we've been using this on the Quill beta for years, so should be good to go. Please let us know if you have any issues.

**Comment by cwhites** [comment]
能使用么？
